src/memfd.rs

```diff
@@ -90,7 +90,7 @@ impl Default for MemfdOptions {
     fn default() -> Self {
         Self {
             allow_sealing: false,
-            cloexec: false,
+            cloexec: true,
```

Comment:
The docstring of `MemfdOptions::new()` should be updated to reflect this.

Reply:
Whoopsie, good point. Updated.